iis-wf/iis-wf-affmatching/src/main/resources/eu/dnetlib/iis/wf/affmatching/oozie_app/workflow.xml

```diff
@@ -0,0 +1,86 @@
+<workflow-app xmlns="uri:oozie:workflow:0.4" name="affiliation-matching">
+
+    <parameters>
+        <property>
+            <name>input_documents_metadata</name>
+            <description>input containing document metadata avro records (eu.dnetlib.iis.metadataextraction.schemas.ExtractedDocumentMetadata)</description>
+        </property>
+        <property>
+            <name>input_organizations</name>
+            <description>input containing organization avro records (eu.dnetlib.iis.importer.schemas.Organization)</description>
+        </property>
+        <property>
+            <name>input_document_to_project</name>
+            <description>input containing document-project relation avro records (eu.dnetlib.iis.referenceextraction.project.schemas.DocumentToProject)</description>
+        </property>
+        <property>
+            <name>input_project_to_organization</name>
+            <description>input containing project-organization avro records (eu.dnetlib.iis.importer.schemas.ProjectToOrganization)</description>
```

Comment:
I would describe what is the purpose of `input_document_to_project` and `input_project_to_organization` because this might not be clear for someone not familiar with InfoSpace internals.